packages/forms/src/coercion.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+
+export interface Coercions {
+  [key: string]: (value: string) => any
+}
+
+export interface CoercionContextInterface {
+  coercions: Coercions
+  setCoercions: React.Dispatch<React.SetStateAction<Coercions>>
+}
+
+const CoercionContext = React.createContext({} as CoercionContextInterface)
```

Comment:
Since we don't have default values for this context, we can do something like this:
```ts
React.createContext<Partial<CoercionContext>>({});
```

https://fettblog.eu/typescript-react/context/#context-without-default-values

Reply:
I don't want to do this. It's effectively make all attributes of the context optional, which I don't want. We actually do have kind of "default values". They're just not set here. They're set further down as props to the Provider. Whatever you put here is never going to be used. It's only for when you use the context outside of its Provider, which we never do.

Reply:
That's fair. We're not trying to satisfy TS here... but rather our intention.